db/seeds.rb

```diff
@@ -9,3 +9,39 @@
 #   Character.create(name: 'Luke', movie: movies.first)
 
 Language.load(Rails.root.join("config/languages.yml"))
+
+def log(*)
+  puts(*) # rubocop:disable Rails/Output
+end
+
+display_name = "admin"
+password = "openstreetmap"
+email = "admin_#{SecureRandom.uuid}@example.com"
+role = "administrator"
+
+admin = User.find_or_create_by!(:display_name => display_name) do |user|
```

Comment:
We should avoid putting non-essential data into the seeds file. This file should only contain the data that's essential to running the site, in all deployments, since the seeds can be run in all environments (test, development and production). These seeds can end up in production databases, perhaps inadvertently, depending on which commands a sysadmin users to set up the database.

See https://github.com/openstreetmap/openstreetmap-website/pull/3030#issuecomment-755458958 for more details on this.